### DISCUSSION TITLE
Changing the color of the SpeedDial component

### DISCUSSION THREAD

**Discussion by jordanrwm** [themesberg/flowbite-svelte#635] [Q&A]
I just tried implementing the SpeedDial component in my project and I cannot figure out a way to change the color of the button from blue to a color of my choice. Is this possible in flowbite-svelte?

**Comment by shinokada** [comment]
A good question!
It is not possible at the moment, but I will change it today.

**Reply by jordanrwm** [reply]
Awesome, thank you so much!

**Comment by shinokada** [comment]
This will be added soon.

![image](https://user-images.githubusercontent.com/147320/229243871-42edea63-0a68-4341-884a-4fa226748278.png)

**Comment by shinokada** [comment]
You can update to the latest version now.

**Comment by shinokada** [comment]
The [docs](https://flowbite-svelte.com/components/speed-dial#Colors) is updated.
